### PR TITLE
fix buildconfig trigger describer to handle new style webhooks with secretreferences

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: pkg/oc/cli/describe/helpers.go [hunk 1]
```diff
@@ -167,32 +167,18 @@ func webHooksDescribe(triggers []buildapi.BuildTriggerPolicy, name, namespace st
...
-			webHookTrigger = trigger.GitHubWebHook.Secret
-
-		case buildapi.GitLabWebHookBuildTriggerType:
-			webHookTrigger = trigger.GitLabWebHook.Secret
-
-		case buildapi.BitbucketWebHookBuildTriggerType:
-			webHookTrigger = trigger.BitbucketWebHook.Secret
-
+		case buildapi.GitHubWebHookBuildTriggerType, buildapi.GitLabWebHookBuildTriggerType, buildapi.BitbucketWebHookBuildTriggerType:
 		case buildapi.GenericWebHookBuildTriggerType:
```
Comment (coreydaley) [inline pkg/oc/cli/describe/helpers.go]: Maybe change this to use reflect.TypeOf instead of a switch statement for what is basically one check?

Comment (bparees) [reply pkg/oc/cli/describe/helpers.go]: it's 4 checks.

Comment (bparees) [reply pkg/oc/cli/describe/helpers.go]: and it's not a type check.

Comment (coreydaley) [reply pkg/oc/cli/describe/helpers.go]: touché, i was somehow reading it as switch trigger.(type)
carry on

Comment (bparees) [reply pkg/oc/cli/describe/helpers.go]: yeah, well you fooled me into reading it that way too after your comment until i reread it :)

**Pull request merged**